pkg/handlers/sdk.go

```diff
@@ -23,40 +23,26 @@ type Handler struct {
 }
 
 // Creates a new handler wrapping the SDK, which is authenticated through the cached client-credentials flow tokens
-func New(platformEndpoint, clientID, clientSecret string, insecure bool) (Handler, error) {
+func New(platformEndpoint, clientID, clientSecret string, insecure, plaintext bool) (Handler, error) {
 	scopes := []string{"email"}
 
 	opts := []sdk.Option{
 		sdk.WithClientCredentials(clientID, clientSecret, scopes),
 		sdk.WithTokenEndpoint(TOKEN_URL),
 	}
 
-	// Try an parse scheme out of platformEndpoint
-	// If it fails, use the default scheme of https
-	// There has to be a better way to do this
-	platformURL, err := url.Parse(platformEndpoint)
-	if err != nil {
-		return Handler{}, err
-	}
+	platformEndpoint = stripScheme(platformEndpoint)
 
-	switch platformURL.Scheme {
-	case "http":
+	if plaintext {
 		opts = append(opts, sdk.WithInsecurePlaintextConn())
-		if platformURL.Port() == "" {
-			platformURL.Host += ":80"
-		}
-	case "https":
-		if platformURL.Port() == "" {
-			platformURL.Host += ":443"
-		}
-	default:
-		return Handler{}, errors.New("invalid scheme")
 	}
```

Comment:
@jakedoublev I talked with @strantalis and we're going to do two things:

1. Change `--insecure` to `--tls-no-verify` to make it more explicit
2. Expect URLs for CLI interaction since CLI users may not know about gRPC (and shouldn't concern themselves). Instead we will expect a platform URL and keep the logic to auto detect the port since gRPC expects explicit ports (which is bad UX)